eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -118,6 +118,11 @@ stages:
                     PackageName: ${{artifact.name}}
                     ServiceName: ${{parameters.ServiceDirectory}}
                     ForRelease: true
+              - template: /eng/common/pipelines/templates/steps/verify-restapi-spec-location.yml
```

Comment:
Please add a comment here to specify that this script verifies both TypeSpec and swagger specification are in `main` branch.

Reply:
Added.